AcademyApp/Designables/UIButtonX.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  UIButtonX.swift
+//  AcademyApp
+//
+//  Created by Peter Sevcik on 14/06/2019.
+//  Copyright © 2019 Peter Sevcik. All rights reserved.
+//
+
+import UIKit
+
+@IBDesignable
+class UIButtonX: UIButton {
```

Comment:
In common is very good to use comments in code and extensions and // MARK to split class code in smaller nicer pieces

Example
`
// MARK: - View controller for  login screen
final class LoginViewController {
     // IBOutlets

     // UIViewController lifecycle
     override func viewDidLoad() {
    super.viewDidLoad()
    setupStyles()
}

// MARK: - Private setup ui methods
private extension LoginViewController {
func setupStyles() {}
`

smth like this later if you use ctrl+6 u can see structure of file nicer
maybe this approach will be deprecated with new xcode and its file structure feature

Reply:
Added marks to organize code. Updated extensions to be private if needed.